recipes/lrez/build.sh

```diff
@@ -1,23 +1,29 @@
 #!/bin/bash
-export HTSLIB_ROOT=${PREFIX}/lib
-rm -rf build
-mkdir build
-cd build
-cmake ..
-make -j2
-cd ..
-
-mkdir -p $PREFIX/bin/
 
+export LDFLAGS="${LDFLAGS} -L${PREFIX}/lib"
+export LIBS="-lz -lm -lc -lboost_iostreams"
+export CPLUS_INCLUDE_PATH=${PREFIX}/include
```

Comment:
The need for this indicates a bug in the source.
```
17:11:05 BIOCONDA INFO (OUT) In file included from src/subcommands/indexFastq.cpp:4:
17:11:05 BIOCONDA INFO (OUT) /opt/conda/conda-bld/lrez_1618419880923/work/src/include/gzIndex.h:62:10: fatal error: zlib.h: No such file or directory
17:11:05 BIOCONDA INFO (OUT)    62 | #include "zlib.h"
17:11:05 BIOCONDA INFO (OUT)       |          ^~~~~~~~
```
This is should read `#include <zlib.h>` to have it find the library headers automatically.